assets/scripts/dialogs/__tests__/GeotagDialog.test.js

```diff
@@ -39,11 +47,19 @@ const initialState = {
 
 describe('GeotagDialog', () => {
   it('renders', () => {
-    const wrapper = renderWithReduxAndIntl(<GeotagDialog />, { initialState })
+    const wrapper = renderWithReduxAndIntl(
+      <GeotagDialog renderPopup={true} />,
+      { initialState }
+    )
     expect(wrapper.asFragment()).toMatchSnapshot()
   })
 
-  it('does not allow a location to be confirmed when geocoded data does not have street data', () => {
+  /*
+  I don't think this statement is true
+  assuming 'street' in this case is the street name data coming from Pelias, we DO
+  allow non street-named places to be associated with a streetmix street
+  */
+  it.skip('does not allow a location to be confirmed when geocoded data does not have street data', () => {
```

Comment:
The `street` property in Redux state is the actual street data of the street, not any geodata from Pelias. Either the description string or the test implementation might be wrong if it led you to think this was Pelias data.

We have several options here.

1. Allow the test description to describe our intended functionality, and delete the function contents so that it doesn't continue to confuse us. Skip the test for now instead of updating the component to pass the test of intended functionality.
2. Decide that allowing non-street places is a feature, not a bug, in which case we call this "working as intended" and delete the entire test.

Reply:
My apologies, that note was left outdated. But it's true, I do often confuse 'street' the Streetmix Street with 'street' inside the data returned by Pelias. 

I'm inclined to lean towards 2 for a couple reasons. I think we'd need to add additional functionality to the geotag dialog to even make this work, and it feels more like a separate concern that would be handled by form/data validation. I also don't know when the edge case could even occur when you'd be using the getotag dialog but somehow not be associating with a street?